modules/ROOT/pages/dataweave-formats-xml.adoc

```diff
@@ -624,6 +650,157 @@ The output represents the transformed XML elements into JSON.
 ]
 ----
 
+[[example12]]
+=== Example: Read `DTD` value
```

Comment:
TODO: edit in source

* backticks around DTD not necessary
* check capitalization of DOCType; prob should match the XML

Reply:
done

Reply:
fixed looks right in HTML